test/features/major_test.js

```diff
@@ -0,0 +1,135 @@
+import webdriver from 'selenium-webdriver'
+import { usingSelenium, By, until } from '../selenium_helpers'
+import { withUsersInTheDatabase } from '../helpers'
+
+const closeTabs = user => {
+  user.getAllWindowHandles()
+    .then(handles => {
+      handles.shift()
+      handles.forEach(handle => {
+        user.switchTo().window(handle)
+        user.close()
+        return handles
+      })
+    })
+}
+
+describe.only('the whole darn app', function(){
+  usingSelenium(function(){
+
+    withUsersInTheDatabase(function(){
+      it('runs through scenario of a player creating a prrr and a coach seeing a new prrr being added and claims it', function(done){
+        this.timeout(60*10000)
+
+        const player = this.createBrowser()
+        const coach = this.createBrowser()
+
+        sinon.stub(Queries.prototype, "getPullRequest").returns(Promise.resolve({FAKE_PR: true}));
+
+        return Promise.resolve()
+          //player logs in
+          .then(_ => player.loginAs(15825329))
+          //player goes to homepage
+          .then(_ => player.visit('/'))
+          //coach logs in
+          .then(_ => coach.loginAs(47313))
+          //coach goes to homepage
+          .then(_ => coach.visit('/'))
+          .then(_ => player.sleep(1*1000))
+          //player clicks add Prrr button, then redirects to addPrrr page
+          .then(_ => player.wait(until.elementLocated(By.className('Navbar-button-add-prrr')), 2000).click())
+          //player pastes pull request url into text box
+          .then(_ => player.findElement(By.className('RequestReviewPage-InputBox'))
+            .sendKeys('https://github.com/AbrahamFergie/Obeisant-Gecko/pull/6'))
+          .then(_ => player.sleep(1*1000))
+          //player once seeing fields are populated appropriately, then clicks add
+          .then(_ => player.findElement(By.className('RequestReviewPage-Button')).click())
+          .then(_ => coach.sleep(1*5000))
+          //player notices that current pull request is under my requested prrrs with pull request address and request by information and opens links
+          .then(_ => player.findElement(By.xpath("//td[@class='PrrrsTable-pr']//a[@href='https://github.com/AbrahamFergie/Obeisant-Gecko/pull/6']")).click())
+          .then(_ => player.findElement(By.xpath("//td[@class='PrrrsTable-requested']//a[@href='https://github.com/AbrahamFergie']")).click())
+          .then(_ => player.sleep(1*2000))
+          //closes popup tabs
+          .then(_ => player.getAllWindowHandles().then(function (handles) {
+            player.switchTo().window(handles[2])
+            player.close()
+            player.switchTo().window(handles[1])
+            player.close()
+            player.switchTo().window(handles[0])
+          }))
+          //coach upon seeing that a Prrr has been placed in queue, then clicks button to claim a Prrr
+          .then(_ => coach.wait(until.elementLocated(By.className('Button-default ReviewButton')), 2000).click())
```

Comment:
I wish we could use real CSS selectors instead of space separated classnames. So id prefer `By.className('Button-default ReviewButton')` to be something like By.css('.Button-default.ReviewButton')`